docker-dist/Dockerfile

```diff
@@ -19,29 +19,22 @@ FROM jboss/wildfly:10.1.0.Final
 
 MAINTAINER Heiko W. Rupp <hawkular-dev@lists.jboss.org>
```

Comment:
Yes, I'll modify the other Dockerfiles while I'm on it. Is that ok? Or just this file?